<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logon</class>
 <widget class="QWidget" name="Logon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log on</string>
  </property>
  <property name="windowIcon">
   <iconset resource="D:/python projects/pyside2 practice/Logon.qrc">
    <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelId">
        <property name="text">
         <string>&amp;Id:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditId</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditId"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPW">
        <property name="text">
         <string>&amp;Password:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditPW</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPW">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOk">
        <property name="text">
         <string>Ok</string>
        </property>
        <property name="icon">
         <iconset resource="D:/python projects/pyside2 practice/Logon.qrc">
          <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditId</tabstop>
  <tabstop>lineEditPW</tabstop>
  <tabstop>buttonOk</tabstop>
 </tabstops>
 <resources>
  <include location="D:/python projects/pyside2 practice/Logon.qrc"/>
 </resources>
 <connections/>
</ui>
